.codeclimate.yml

```diff
@@ -27,29 +25,28 @@ engines:
         ruby:
           mass_threshold: 25
           count_threshold: 5
-        javascript:
+        javascript: 
   eslint:
     enabled: false
-    channel: "eslint-3"
+    channel: eslint-3
   fixme:
-    # let's enable later
     enabled: false
   markdownlint:
-    # let's enable later
     enabled: false
   rubocop:
     enabled: true
-    config: '.rubocop_cc.yml'
+    config: ".rubocop_cc.yml"
+    channel: rubocop-0-69
```

Comment:
@djberg96 Just making sure... Is this the only non-cleanup change to the file?

Reply:
Yeah, the rest is just the format that `YAML.dump` gives us.

Reply:
OK, good. Are the codeclimate failures the expected failures we discussed yesterday? If so, I'll merge it and cut a new version of the GEM.

Reply:
Yes. Not sure why you need to cut a new version of the gem, though, since it only affects our repo.